CHANGELOG.md

```diff
@@ -117,10 +132,30 @@
 [porges]:
   https://github.com/porges
 
+[300]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/300
+[288]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/288
+[282]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/282
+[277]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/277
+[269]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/269
+[266]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/266
+[260]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/260
 [255]:
   https://github.com/hedgehogqa/fsharp-hedgehog/pull/255
+[252]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/252
 [250]:
   https://github.com/hedgehogqa/fsharp-hedgehog/pull/250
+[247]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/247
+[244]:
+  https://github.com/hedgehogqa/fsharp-hedgehog/pull/244
```

Comment:
@adam-becker, here I'm adding each one of the PR URLs from above. So far this is done manually.

Reply:
@moodmosaic Hopefully putting the issues into a milestone makes this part of the release easier.